main/ertp/ERTP-Guide/Overview.md

```diff
@@ -0,0 +1,310 @@
+  is used to validate `payments` received from untrusted parties. Has a one-to-one relationship
+  with both a `brand` and a `mint`. 
+- **[AmountMath](./AmountMath.md)** ([glossary](https://agoric.com/documentation/glossary/#amountmath)):
+  Methods to do math operations on `amounts`. Each `brand` has its own `amountMath` in a one-to-one relationship.
+
+![ERTP object relationships](./assets/relationships1.svg) 
+
+Let's look at an example. Suppose there is the "Quatloos" `brand`. That means there is also:
+- A "Quatloos `mint`" that is the only `mint` that can create new Quatloos assets.
+- A "Quatloos `issuer`" that is the only `issuer` that can create a new `purse` to contain Quatloos and 
+  operate on a `payment` containing Quatloos.
+- A "Quatloos `amountMath`" that is the only `amountMath` whose operations work on an `amount` whose `brand` is Quatloos.
+
+![ERTP object relationships 2](./assets/relationships2.svg) 
+
+We've already mentioned our final two components:
+- **[Purse](./PursesAndPayments.md)** ([glossary](https://agoric.com/documentation/glossary/#purse)): An
+  object for holding digital assets of a specific `brand`.
+- **[Payment](./PursesAndPayments.md)** ([glossary](https://agoric.com/documentation/glossary/#payment)):
+  An object for transfering digital assets of a specific `brand` to another party.
+  
+Similar to other component instances, a `purse` and a `payment` only work with one
+`brand`. So a `purse` or `payment` that holds Quatloos cannot hold an asset of `brand` Moola or vice versa. 
+You cannot change the `brand` a `purse` or `payment` was originally associated with. Once you create a
+"Quatloos `purse`" or "Quatloos `payment`", they can never hold anything other than Quatloos.
+
+However, unlike the other components, these are not one-to-one relationships. There can be thousands or more
+`purses` or `payments` that hold Quatloos or any other `brand`.
+
+## Life of Assets
+
+Let's look at some asset "lifecycles". While it's very rare for an asset to be destroyed, these "lifecycles"
+show assets from their creation through common usage patterns. These are deliberately stripped down to their
+basic, core, functionality. Optional parameters and non-core operations are not shown, nor are some significant
+concepts which would make this introduction more confusing. These are covered on the component-specific pages.
+
+### Asset creation and storage
+
+![Asset creation](./assets/assetCreation.svg)
+```js
+const { quatloosIssuer, quatloosMint, quatloosAmountMath, quatloosBrand } = makeIssuerKit('quatloos');
+```
+First, you take a string naming a new to the system `brand` and use it as the argument to
+`makeIssuerKit()`. The name underestimates its power; in addition to returning a new `issuer`
+for the specified `brand`, it also returns a new `mint`, `amountMath`, and formal `brand` 
+for the argument. All are in one-to-one associations with each other. 
```

Comment:
Good idea about the naming scheme, but I don't want to bury it in an otherwise unrelated section (or have it take attention away from what the section's actually about). I moved that to its own section just before this one, and will include it in an appropriate place in Zoe docs.